k8s/base-config.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: v1
+kind: Namespace
```

Comment:
Missing default network traffic filtering rule for this Namespace. This is best practice to deny all internet traffic by default and explicitly allow specific desired traffic. 
According to Owasp 10 this falls under Security misconfiguration and may allow attackers to discover your resources and attempt exploitation on other vulnerabilities to gain access to your environment. 
Suggestions: please specify a default policy to be used blocking all external traffic and allowing only the necessary one.